### DISCUSSION TITLE
erro ao renderizar o cheoutBrick

### DISCUSSION THREAD

**Discussion by Fernando224-por** [mercadopago/sdk-react#95] [Q&A] [answered]
estuy intentando renderizar el checkoutBrick tipo Payment en mi proyecto pero tengo este error en particular
![image](https://github.com/mercadopago/sdk-react/assets/110569835/6de6e1bd-c14b-4718-ac18-73f91ac0e805)

Aqui está uma breve imagem do código
![image](https://github.com/mercadopago/sdk-react/assets/110569835/bb3d8ae1-b9a2-4e5d-aeb5-1c737129b787)

Para definir os valores de amount e preferenceId, estou usando o manipulador de estado Zustand

**Comment by wilsntn** [comment, marked as answer]
Voce esta usando o access token e nao a chave publica
